socorro-cmd

```diff
@@ -93,7 +93,7 @@ COMMANDS = [
     Group(
         'Miscellaneous', {
             'showcommands': showcommands_cmd,
-            'signature': 'socorro.signature.__main__.main'
+            'signature': 'socorro.signature.cmd_signature.main'
```

Comment:
I moved the command code so it jives with extracting to an external library better. This does break the whole:

```
python -m socorro.signature ...
```

thing, but that's ok.